packages/map-popup/src/MapPopup.story.tsx

```diff
@@ -85,7 +85,7 @@ export const FloatingVehicleEntity = (): JSX.Element => (
 );
 
 export const StopEntityWithFocusTrap = (): JSX.Element => (
-  <FocusTrapWrapper id="storybook-stop" closePopup={() => {}}>
+  <FocusTrapWrapper closePopup={() => {}} id="storybook-stop">
```

Comment:
Can `closePopup` be optional with a null check in `FocusTrapWrapper`?

Reply:
Added in [c24554b](https://github.com/opentripplanner/otp-ui/pull/739/commits/c24554b92f8787e55193770916ef367642edf08e), let me know if this isn't what you meant